src/express/middlewares/img-storage.js

```diff
@@ -0,0 +1,19 @@
+const multer = require(`multer`);
+const path = require(`path`);
+const {nanoid} = require(`nanoid`);
+
+const UPLOAD_DIR = `../upload/img/`;
+
+const uploadDirAbsolute = path.resolve(__dirname, UPLOAD_DIR);
+
+const storage = multer.diskStorage({
+  destination: uploadDirAbsolute,
+  filename: (req, file, cb) => {
+    const uniqueName = nanoid(10);
+    const extension = file.originalname.split(`.`).pop();
+    console.log('file', file)
+    cb(null, `${uniqueName}.${extension}`);
+  }
+});
+
+module.exports.storage = storage;
```

Comment:
missing EOL symbol https://stackoverflow.com/questions/39525417/visual-studio-code-how-to-show-line-endings